homeassistant/components/flipr/__init__.py

```diff
@@ -69,6 +81,32 @@ def find_duplicate_entries(entries):
     entries = hass.config_entries.async_entries(DOMAIN)
 
     if find_duplicate_entries(entries):
+        ir.async_create_issue(
+            hass,
+            DOMAIN,
+            "duplicate_config",
+            breaks_in_ha_version="2024.10.0",
+            is_fixable=False,
+            severity=ir.IssueSeverity.ERROR,
+            translation_key="duplicate_config",
+        )
+
         raise ConfigEntryError(
             "Duplicate entries found for flipr with the same user email. Please remove one of it manually. Multiple fliprs will be automatically detected after restart."
         )
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate config entry."""
+    _LOGGER.debug("Migration of flipr config from version %s", entry.version)
+
+    if entry.version == 1:
+        # In version 1, we have flipr id as config entry unique id and one device per config entry.
```

Comment:
```suggestion
        # In version 1, we have flipr device id as config entry unique id and one device per config entry.
```
Maybe it helps that we explain that it used to be device specific